<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exam</class>
 <widget class="QWidget" name="exam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>在线考试</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>在线考试</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>科目</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leExamName">
      <property name="text">
       <string>通信原理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>考试时间</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leExamTime">
      <property name="text">
       <string>45</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>分钟</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>剩余时间</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leLeftTime">
      <property name="text">
       <string>45</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="leQuestionType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>测试题</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teQuestion">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>431</width>
     <height>221</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>题型：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbxSub_3">
      <item>
       <property name="text">
        <string>选择题</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>判断题</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>填空题</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>题号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>第</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leQuestionID">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>题</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>181</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>您的答案</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leMyAns">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>下一题</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrevious">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>上一题</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>181</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>参考答案</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leChkAns">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSubmit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>交卷</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>消息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnchkAn">
   <property name="geometry">
    <rect>
     <x>496</x>
     <y>240</y>
     <width>130</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
